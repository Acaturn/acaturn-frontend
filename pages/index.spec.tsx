import React from 'react';
import { shallow } from 'enzyme';
import Index from './index';

describe('Index', () => {
  it('should render index page', () => {
    const index = shallow(<Index />);
    expect(index).toMatchSnapshot();
  });
});
