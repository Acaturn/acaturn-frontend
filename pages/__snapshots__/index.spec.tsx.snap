// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Index should render index page 1`] = `ShallowWrapper {}`;
